### PR TITLE
Feature/search input

### LINKED ISSUE #13: Change state of UI to display query results

### LINKED ISSUE #14: Add search query functionality

### PR DESCRIPTION
**Pull request by Max9545**
# Selector FE PR

## What does this merge do? Check all that apply.
- [x] Adds a new feature
- [ ] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## Summary of changes

- Make the search input render the album of a users choice

## Notes for reviewer (any known bugs, areas of interest, etc.)

- See if onSubmit makes any difference?

## Next Steps (Future Iterations)

- Add error handling to search
- Need Cypress Tests

### CODE REVIEW

**Review by monshan — APPROVED** [review]
Everything looks good to go as we've covered over the pair-programming this morning.

File: src/Components/Search/Search.js [hunk 1]
```diff
@@ -11,17 +11,24 @@ const ALBUM_TITLES = gql`
...
+
+  //   return data.map(title => {
+  //     return <option value={ title } />
+  //   })
+  // }
 
-    return data.map(title => {
-      return <option value={ title } />
-    })
+  const helpSetSearch = (event) => {
```
Comment (rdtho2525) [inline src/Components/Search/Search.js:29]: As discussed, we should update this function identifier to be more descriptive and action-oriented.

**Review by rdtho2525 — APPROVED** [review]
As expected, the dashboard responds appropriately to the input submitted in the search field. All other functionality and design is maintained, as well.

File: src/Components/Search/Search.js (hunk 1 above)
Comment (Max9545) [reply src/Components/Search/Search.js:29]: will brainstorm

**Pull request merged**